Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "DeepDiffs"
+uuid = "ab62b9b5-e342-54a8-a765-a90f495de1a6"
+license = "MIT"
+desc = "Compute and pretty-print diffs for data structures like arrays and dictionaries"
+keywords = ["diff"]
+version = "0.1.0"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```

Comment:
Most likely `Compat` was included to support Julia 1.0 when this package supported Julia 0.6. I would try removing this dependency and the `using Compat` calls and verify it is still needed.

Reply:
Addressed